### PR TITLE
[CN-1148] Init UCN tutorial

### PR DESCRIPTION
**Pull request by semihbkgr**

### CODE REVIEW

File: docs/modules/ROOT/examples/operator-user-code-namespace/hazelcast.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast
```
Comment (kutluhanmetin) [inline docs/modules/ROOT/examples/operator-user-code-namespace/hazelcast.yaml:4]:
```suggestion
  name: my-hazelcast
```
?

**Review by SeriyBg — APPROVED** [review]

**Comment by kutluhanmetin** [comment]
@semihbkgr AFAIK, we can use workload identity and service account in UCN as well. Shall we put a reference for it: https://docs.hazelcast.com/operator/latest-snapshot/authorization

File: docs/modules/ROOT/examples/operator-user-code-namespace/hazelcast.yaml (hunk 1 above)
Comment (semihbkgr) [reply docs/modules/ROOT/examples/operator-user-code-namespace/hazelcast.yaml:4]: The name of the resource should be `hazelcast`, as indicated by the field in the map resource: `spec.hazelcastResourceName: hazelcast`.

**Comment by semihbkgr** [comment]
@kutluhanmetin  I think it is better to keep it simple by focusing only on the UCN.

**Review by kutluhanmetin — APPROVED** [review]

**Pull request merged**